styleguide/source/assets/scss/00-base/_typography.scss

```diff
@@ -59,3 +57,78 @@ small,
   @include font-size($small-font-sizes);
   font-style: italic;
 }
+
+strong,
+.ama__strong,
+%ama__strong {
+  @include type($myriad-pro, $font-weight-bold);
+  @include font-size($p-font-sizes);
+}
+
+// Typography for topic pages
+body.topic {
+  /* stylelint-disable property-no-vendor-prefix */
+  -webkit-text-size-adjust: 100%;
+  -webkit-font-smoothing: antialiased;
+  /* stylelint-enable */
+  color: $black;
+  @include type($myriad-pro, $font-weight-regular);
+  @include font-size($p-topic-font-sizes);
+  //Topic Headers
+
+  h1,
+  .topic__h1,
+  %topic__h1 {
+    @include type($kepler-std, $font-weight-semibold);
+    @include font-size($h1-topic-font-sizes);
+  }
+
+  h2,
+  .topic__h2,
+  %topic__h2 {
+    @include type($kepler-std, $font-weight-semibold);
+    @include font-size($h2-topic-font-sizes);
+  }
+
+  h3,
+  .topic__h3,
+  %topic__h3 {
+    @include type($kepler-std, $font-weight-semibold);
+    @include font-size($h3-topic-font-sizes);
+  }
+
+  h4,
+  .topic__h4,
+  %topic__h4 {
+    @include type($myriad-pro, $font-weight-bold);
+    @include font-size($h4-topic-font-sizes);
+  }
+
+  h5,
+  .topic__h5,
+  %topic__h5 {
+    @include type($myriad-pro, $font-weight-bold);
+    @include font-size($h5-topic-font-sizes);
+  }
+
+  h6,
+  .topic__h6,
+  %topic__h6 {
+    @include type($myriad-pro, $font-weight-bold);
+    @include font-size($h6-topic-font-sizes);
+  }
+
+  small,
+  .topic__type-small,
+  %topic__type-small {
```

Comment:
these should be doubly hyphen before small since it's a modifier (e.g `topic__type--small`)